### PR TITLE
Added unit test for the API endpoints

### PR DESCRIPTION
**Pull request by Daniel-olaO**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def sendText(number):
                 to=number,
                 from_=os.environ.get('TWILIO_PHONE_NUMBER'),
                 body= f'\nToday\'s NASA Astronomy Picture of the Day is: {data["title"]}.\n\n{data["explanation"]}',
-                media_url=data['hdurl'] if 'hdurl' in data else data['url']
+                media_url=data['url']
```
Comment (Russell-Pollari) [inline api/views.py:163]: Was this the fix for the image size?

File: api/tests.py [hunk 2]
```diff
@@ -1,3 +1,66 @@
-from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from .models import User
 
-# Create your tests here.
+class UserTests(APITestCase):
```
Comment (Russell-Pollari) [inline api/tests.py:6]: cool!

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]
Looks good. Did not test locally though. 

Just have one quick question

File: api/views.py (hunk 1 above)
Comment (Daniel-olaO) [reply api/views.py:163]: Yep

**Review by Russell-Pollari — APPROVED** [review]
Next time, the fix for the image bug should be in a separate PR with an appropriate title. I had to guess at the line

**Pull request merged**